app/models/task.rb

```diff
@@ -0,0 +1,11 @@
+class Task < ActiveRecord::Base
+  belongs_to :person
+
+  def self.count_completed(person_id)
+    where(person_id: person_id).where.not(completed_at: nil).count
+  end
+
+  def self.count_not_completed(person_id)
+    where(person_id: person_id).where(completed_at: nil).count
+  end
```

Comment:
I'm a little confused what count_complteted is doing between the person and task models.